### DISCUSSION TITLE
Ruuvitag to mqtt

### DISCUSSION THREAD

**Discussion by spueli99** [arendst/Tasmota#22680] [Q&A]
Hello everyone, I guess I need some help. I have several ruuvitag and would like to read them out via tasmota and transfer the data via mqtt. Is this possible? Rudimentary knowledge is available. I use several tasmota reading heads to read the electricity meters. Unfortunately, an esp32 with tasmota ble cannot find the ruuvitag.
I would also like to use ruuvi under tasmota, publish the data via mqtt and integrate it into home assistant. As a test, I tried OpenMQTTGateway, it works perfectly. I would be very grateful for any support. Unfortunately I can't find any further information about this.

**Comment by Jambola77** [comment]
Official Documentation can be found her: https://docs.ruuvi.com/
And sample projects here: https://github.com/ruuvi
